src/Iot.Ingestion.RouterService/RouterService.cs

```diff
@@ -293,18 +272,15 @@ protected override async Task RunAsync(CancellationToken cancellationToken)
                         eventHubPartitionId,
                         DateTime.UtcNow);
 
-                    eventHubReceiver =
-                        await
-                            eventHubClient.GetDefaultConsumerGroup()
-                                .CreateReceiverAsync(eventHubPartitionId, DateTime.UtcNow, newEpoch);
+                    eventHubConsumerClient = new EventHubConsumerClient(consumerGroup, connectionString, eventHubName);
                 }
 
                 // epoch is recorded each time the service fails over or restarts.
                 await epochDictionary.SetAsync(tx, "epoch", newEpoch);

```

Comment:
This doesn't make sense in context;  the client is being created and no owner level (epoch) is in use.   The majority of this method is pretty much a no-op.